state/state.go

```diff
@@ -8,14 +8,18 @@ import (
 	"github.com/pganalyze/collector/config"
 )
 
+type DBStats struct {
```

Comment:
Maybe `SchemaStats` is a better name? (there are other statistics about the database that are not inside this struct, e.g. statements could also be considered to be part of a database)